spec/peep_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'peep'
+require 'date'
+
+describe Chitter do
+  describe '.all' do 
+    it 'returns all peeps' do 
+      ENV['ENVIRNOMENT'] = 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+      connection.exec("INSERT INTO peeps (message) VALUES ('test_peep_1') ;")
+      connection.exec("INSERT INTO peeps (message) VALUES ('test_peep_2') ;")
+      connection.exec("INSERT INTO peeps (message) VALUES ('test_peep_3') ;")
+
+      peeps = Peep.all 
+
+      expect(peeps).to include("test_peep_1 - " + Date.today.to_s)
+      expect(peeps).to include("test_peep_2 - " + Date.today.to_s)
+      expect(peeps).to include("test_peep_3 - " + Date.today.to_s) 
```

Comment:
A stronger test would be to use `eq` instead of `include`. With `eq`, you could compare the whole array at once and on top of that, you can make sure it doesn't contain anything _other_ than what you expect.